strings-csharp/GetFirstNCharactersOfAString/GetFirstNCharactersOfAString/FirstNCharactersOfStringGetter.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Text;
+
+namespace GetFirstNCharactersOfAString;
+public class FirstNCharactersOfStringGetter(string inputString = "CodeMaze", int numberOfCharacters = 4)
+{
+    private readonly string _inputString = inputString;
+    private readonly int _numberOfCharacters = numberOfCharacters;
```

Comment:
Since we are using a primary constructor, we don't need these member variables, we can just use the parameters from the constructor